### PR TITLE
dont loop through empty modules path

### LINKED ISSUE #1012: Webpack Production Build: paths is not iterable
During booting of the app we get an error ` n.paths is not iterable` , which points to this function below 

```
// Determine the project root
// https://stackoverflow.com/a/18721515
function getPkgJsonDir() {
    console.info("module", module)
    for (const modPath of module.paths) {    
```

module is {` children: [], exports: [Setter] } ` and seems to be missing the  `paths key`

### PR DESCRIPTION
**Pull request by lukepolo**
if the modules paths is empty don't loop through them fixes #1012

### CODE REVIEW

**Comment by lukepolo** [comment]
PR : https://github.com/taskforcesh/bullmq/pull/1013


before: 

image


after; 

image

File: src/commands/script-loader.ts [hunk 1]
```diff
@@ -484,6 +484,9 @@ async function getFilenamesByPattern(pattern: string): Promise<string[]> {
 // Determine the project root
 // https://stackoverflow.com/a/18721515
 function getPkgJsonDir(): string {
+  if(!module.paths) {
```
Comment (roggervalf) [inline src/commands/script-loader.ts:487]: what about:
`for (const modPath of (module.paths||[])) {`

Comment (lukepolo) [reply src/commands/script-loader.ts:487]: Sure that works too

Comment (lukepolo) [reply src/commands/script-loader.ts]: making sure this still works as expected

**Comment by roggervalf** [comment]
Thanks @lukepolo, it's released

**Pull request merged**